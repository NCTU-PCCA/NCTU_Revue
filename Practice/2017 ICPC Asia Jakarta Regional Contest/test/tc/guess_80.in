190534452996 38
>><>>>>><<<>>>>><><<<<><>><<<><>>><<>=
