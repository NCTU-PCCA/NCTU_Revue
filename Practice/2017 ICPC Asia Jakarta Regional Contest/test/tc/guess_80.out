95267226498 142900839747 166717646372 154809243059 160763444715 163740545543 165229095957 165973371164 166345508768 166159439966 166066405565 166019888364 166043146964 166054776264 166060590914 166063498239 166064951902 166064225070 166064588486 166064406778 166064315924 166064270497 166064247783 166064259140 166064253461 166064256300 166064257720 166064257010 166064256655 166064256477 166064256566 166064256521 166064256543 166064256554 166064256560 166064256557 166064256555 166064256556
